### PR TITLE
add assignee_name filter for ContractRequest list field

### PR DESCRIPTION
**Pull request by kiurchv**

### CODE REVIEW

File: apps/graphql/lib/graphql_web/resolvers/contract_request.ex [hunk 1]
```diff
@@ -24,6 +25,39 @@ defmodule GraphQLWeb.Resolvers.ContractRequest do
...
+    |> order_by(^order_by)
+  end
+
+  defp prepare_filter([]), do: []
+
+  defp prepare_filter([{:assignee_name, value} | tail]) do
+    assignee_ids =
+      Employee
+      |> join(:inner, [e], p in assoc(e, :party))
+      |> where([e], e.employee_type == "NHS")
```
Comment (progsmile) [inline apps/graphql/lib/graphql_web/resolvers/contract_request.ex]: Does only NHS can perform search?

File: apps/graphql/lib/graphql_web/resolvers/contract_request.ex [hunk 2]
```diff
@@ -24,6 +25,39 @@ defmodule GraphQLWeb.Resolvers.ContractRequest do
...
+      |> where(
+        [..., p],
+        fragment(
+          "to_tsvector(concat_ws(' ', ?, ?, ?)) @@ to_tsquery(?)",
```
Comment (progsmile) [inline apps/graphql/lib/graphql_web/resolvers/contract_request.ex]: Why `p.last_name` isn't first among `first_name`, `second_name` params? Is it better than `like` search?

Comment (kiurchv) [reply apps/graphql/lib/graphql_web/resolvers/contract_request.ex]: In the current case, word order makes not so much sense. All lexemes in the `tsvector` will have the same priority, but we can place `last_name` first  assuming that most of the queries will start from last name

File: apps/graphql/lib/graphql_web/resolvers/contract_request.ex (hunk 1 above)
Comment (kiurchv) [reply apps/graphql/lib/graphql_web/resolvers/contract_request.ex]: Nope, but we can assign only NHS employee to contract request

**Pull request merged**